### DISCUSSION TITLE
How do I use strings in a smart contract?

### DISCUSSION THREAD

**Discussion by br0wnD3v** [MinaProtocol/mina#12998] [Q&A]
Hey everyone, Recently started working with zkApps and not sure about how do we set 'strings' in contract storage that I can call later if I want to.

Say I want to set a variable message as 
```
@method init() {
    message = "Hello World";
}
```
And return it back later as
```
@method getMessage(){
   return this.message;
}
```
Any help is appreciated, thanks :-)

**Comment by br0wnD3v** [comment]
Found something called `CircuitString.from('some string');`

**Comment by ymekuria** [comment]
@br0wnD3v checkout the Mina zkApp developers and zkApp questions [discord channel](https://discord.com/channels/484437221055922177/@home). You can find answers to common questions and get help from the community there.
